### DISCUSSION TITLE
有 http3 c/c++的 示例吗？

### DISCUSSION THREAD

**Discussion by yaomoshen520** [Tencent/tquic#353] [Q&A (问题和回答)]
对c/c++不是太熟， 是否有 http3的示例呢？

**Comment by iyangsj** [comment]
目前有rust版本 http/3示例，参见https://github.com/Tencent/tquic/tree/develop/tools/src/bin
后续版本会增加c/c++版本的http/3示例

**Reply by maxfong** [reply]
期待早日提供c/c++版本的http/3示例
